incorporate Azalee's suggested text (episode 5) (#29)
datacarpentry/astronomy-python: _episodes/05-join.md

To do that we will:

1. Use the `JOIN` operator to look up each Pan-STARRS `obj_id` for the stars 

> we are interested in in the` panstarrs1_best_neighbour` table using the `source_id`s

Author: I think the double "in" is correct here. We're looking for the stars we are interested in in the table.

## Joining tables

Now let's get to the details of performing a `JOIN` operation.

We are about to build a complex query using software that doesn’t provide us with much information on debugging.
For this reason we are going to start with a simplified version of what we want to do until we’re sure we’re joining

> the tables correctly, then we’ll slowly add more layers of complexity, checking at each stage that our query still works.

Reviewer: ```suggestion
the tables correctly, then we’ll slowly and add more layers of complexity, checking at each stage that our query still works.
```

Now that there's more than one table involved, we can't use simple
column names any more; we have to use **qualified column names**.
In other words, we have to specify which table each column is in. 

> The column names do not have to be the same and in fact in the next join they will not be. 

Reviewer: ```suggestion
The column names do not have to be the same and, in fact, in the next join they will not be. 
```

That is one of the reasons that we explicitly specify them.
Here's the complete query, including the columns we want from the Gaia
and best neighbor tables. Here you can start to see that using the abbreviated names

> is making our query easier to read and is less typing for us.

Reviewer: ```suggestion
is making our query easier to read and requires less typing for us.
```

## Transforming coordinates

> As always we retrieve coordinates from the database in ICRS coordinates but we always want to visualize them in GD-1 coordinates. 

Reviewer: ```suggestion
As always we retrieve coordinates from the database in ICRS coordinates but we always want to visualize them in GD1 coordinates. 
```

I notice we are mixing GD-1 and GD1, is there a reason or should be make it consistent?
